### PR TITLE
correct role handling and S3FS bypass

### PR DESCRIPTION
**Pull request by ErinWeisbart**
Correct credential handling to enable AWS IAM role usage.

### CODE REVIEW

**Comment by ErinWeisbart** [comment]
Closes #132 by changing load_data_csv access to download.
Closes #114.

**Comment by ErinWeisbart** [comment]
Noting that S3FS bypass appears to be working fine and I had several tests of all the files downloading correctly and then apparently stochastically one run threw:
`OSError: [Errno 30] Read-only file system: '/home/ubuntu/local_input/path/to/image/cdp2bioactives_a22_s4_w33236d730-2143-4738-bfac-685f917a717d.tif.2FE98647'`

**Comment by ErinWeisbart** [comment]
Full S3FS bypass is working at this point.
Still haven't finished necessary changes to run-worker.sh to make it mount S3 bucket using new credential handling.

**Comment by ErinWeisbart** [comment]
I have this branch running on a dataset now and it works (!), successfully completely bypassing S3FS and using a role.

File: files/exampleJob.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-  "_comment1": "Paths in this file are relative to the root of your S3 bucket",
-  "pipeline": "projects/analysis.cppipe", 
-  "data_file": "projects/list_of_images.csv", 
+  "_comment1": "Paths in this file are relative to the root of S3 buckets",
+  "_comment2": "pipeline, data_file, and input are relative to INPUT_BUCKET; output to OUTPUT_BUCKET",
```
Comment (bethac07) [inline files/exampleJob.json:3]: Should this be source and dest?

**Review by bethac07 — APPROVED** [review]
Overall, this seems great, and super exciting that it's working! Thank you so much 

For future, tiiiiiny nitpick - would be great to not mix in a ton of stylistic changes (like `"`->`'` in a PR like this already making big behavior changes, or at least make those an explicit commit, just to a) keep the size and readability of the PR reasonable and b) assuming commits aren't squashed, by having them be an explicit commit, you see that that's why a line was changed ("oh, that was done for stylistic reasons" as opposed to "wait, what does this have to do with role handling?"). For this one, there are a lot of commits, so up to you if you'd like us to squash or not, but just something small for the future.

Congrats on this, it was a huge achievement!

File: files/exampleJob.json (hunk 1 above)
Comment (ErinWeisbart) [reply files/exampleJob.json:3]: yes, good catch.

**Comment by ErinWeisbart** [comment]
Thanks @bethac07 ! 
Sorry, this PR kinda turned into a chaos of commits in all the troubleshooting. I'm not honestly sure how a lot of stylistic changes ended up being mixed in, but I will certainly try harder to keep them to their own commit in the future.

**Pull request merged**